sandy/fy.py

```diff
@@ -404,11 +404,11 @@ def custom_perturbation(self, zam, mt, e, zap, pert):
         e : `float`
             Energy of the fissioning system to which which perturbations
             are to be applied.
-        zap : `int`
-            ZAP of the product to which perturbations are to be
-            applied.
-        pert : `float`
-            Perturbation coefficients as ratio values.
+        zap : `int` or `list`
```

Comment:
I don't like `zap` and `pert` because they are redundant when multiple perturbations are given.
Then, I would rather have `pert` as a list.

Also, the scalar version can be also generalized as a special case of the vector version, with the list containing only one element.

Reply:
I agree it's redundant, so we can keep `pert` only as a `pd.Series` and remove `zap`. `pert` will be a row of the DataFrame `sandy.Samples.data` in the sampling procedure. If `pert` was a list, the `zap` should be passed in ascending order by the user. If you prefer to have `pert` as a list and two parameters, I will implement it and specify in the docstring that the two should be consistently sorted

Reply:
I prefer the list, but it's up to you, considering consistency and maintainability.

Reply:
> I agree it's redundant, so we can keep `pert` only as a `pd.Series` and remove `zap`. `pert` will be a row of the DataFrame `sandy.Samples.data` in the sampling procedure. If `pert` was a list, the `zap` should be passed in ascending order by the user. If you prefer to have `pert` as a list and two parameters, I will implement it and specify in the docstring that the two should be consistently sorted

As i see it, `zap` and `pert` should not be in ascending order, right?! They should just be in a consistent order.

Reply:
Both ways are fine, but if you go for a series, then I would rather have a `dict`, which is equivalent. 
What do you think?

Reply:
My concern is that when filtering `Fy.data` with the list of ZAPs, the index of the resulting sub-DataFrame will be ordered ascendingly independently of the order of the list. This means that the perturbation coefficients should be ordered ascendingly from the one to apply to the nuclide to the lower ZAP. To me, using a data structure that keeps the information together would help. I can agree with the dictionary but the multiplication in line 437 ensures that a perturbation coefficient is multiplied by the fission yield of the corresponding ZAP thanks to the index, so I would have to cast the dictionary to a pd.Series. 

The implementation with the pd.Series is in the next commit